web-common/src/features/dashboards/leaderboard/LeaderboardMeasureSelector.svelte

```diff
@@ -72,9 +89,19 @@
       class="flex flex-row items-center ui-copy-muted"
       style:grid-column-gap=".4rem"
       in:send={{ key: "leaderboard-metric" }}
-      style:max-width="355px"
+      style:max-width="400px"
     >
-      <div class="whitespace-nowrap">Dimension Leaders by</div>
+      <SeachableFilterButton
```

Comment:
Could we possibly line up these selectors with the text in the first leaderboard in the x direction?

<img width="386" alt="Screen Shot 2023-04-11 at 6 00 37 AM" src="https://user-images.githubusercontent.com/95735/231170339-445b6f71-cec0-44a7-b831-71d3f9a978b9.png">